Expensify/App
# Translate predefined pronouns when changing the app's language.

### Details
Check out the conversation in https://github.com/Expensify/App/issues/5113

### Fixed Issues
$ https://github.com/Expensify/App/issues/5113

### Tests
1. Launch the app and log in
2. Click settings > profile
3. Change preferred pronouns to "Call me by name" and save
4. Click the back arrow
5. In preferences change language to Spanish
6. Click profile (Perfil)
7. The selected option should be "Llámame por mi nombre"

### QA Steps
1. Launch the app and log in
2. Click settings > profile
3. Change preferred pronouns to "Call me by name" and save
4. Click the back arrow
5. In preferences change language to Spanish
6. Click profile (Perfil)
7. The selected option should be "Llámame por mi nombre"

### Tested On


### Screenshots


#### Web
https://user-images.githubusercontent.com/6759612/136174329-d392566f-fd0e-4e3f-8a01-3e7b49b10482.mp4

#### Mobile Web
https://user-images.githubusercontent.com/6759612/136179186-101f7513-4e69-4da1-8326-acca08d8b182.mp4

#### Desktop


#### iOS


#### Android
